kms/kms.rb

```diff
@@ -174,18 +174,90 @@
       Cloudkms::CryptoKey.new(purpose: "ENCRYPT_DECRYPT")
   )
 
-  puts "Created version #{crypto_key_version.name} for key " +
-       "#{crypto_key} in key ring #{key_ring_id}"
+  version = crypto_key_version.name.split("/").last
```

Comment:
PHP - does the same thing (https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/kms/api/src/functions.php#L215)
Nodejs - does not. (https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/1225)

I could just revert these changes :)